_posts/2015-02-06-Utilising-API-Blueprint-in-API-Clients.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Utilising API Blueprint in API Clients
+excerpt: Imagine an API client when you specify which resource and action you want to perform from an API Blueprint.
+layout: post
+date: 2015-02-06 18:00:00 +0000
+author: kyle
+published: true
+comments: true
+---
+
+I’m excited to release a new way to interface to API Blueprints. A client-side framework in Swift that allows you to introspect and perform actions at run-time.
```

Comment:
The last sentence here is a sentence fragment. Maybe consider how to tie these two sentences together.